samples/grpc/shopping-analytics-service-java/src/main/java/shopping/analytics/ShoppingCartEventConsumer.java

```diff
@@ -108,7 +105,7 @@ public static void init(ActorSystem<?> system) {
     int numberOfProjectionInstances = 4;
     String projectionName = "cart-events";
     List<Pair<Integer, Integer>> sliceRanges = Persistence.get(system).getSliceRanges(numberOfProjectionInstances);
-    String streamId = "cart";
+    String streamId = system.settings().config().getString("akka.projection.grpc.consumer.stream-id");
```

Comment:
Should we make the `streamId` a public field on GrpcReadJournal, so that even if it needs to be passed to the source provider at least this isn't needed and you know it's the same used for the journal?

Reply:
Det låter som en bra ide. Så parametern till SourceProvider blir `eventsBySlicesQuery.streamId`.

Reply:
lol 🇸🇪

Reply:
😄 added in https://github.com/akka/akka-projection/pull/705/commits/d581fc1cbc84b6f810364a6f70dc225d09f514c4